src/highlight/__tests__/HighlightCanvas-test.tsx

```diff
@@ -56,5 +56,11 @@ describe('HighlightCanvas', () => {
             const wrapper = getWrapper();
             expect(wrapper.find('canvas').hasClass('ba-HighlightCanvas')).toBe(true);
         });
+
+        test('should render, excluding 0 width and height values', () => {
+            const wrapper = getWrapper();
+            wrapper.setProps({ shapes: [mockRect, noWidthRect, noHeightRect] });
+            expect(wrapper.find('canvas').hasClass('ba-HighlightCanvas')).toBe(true);
```

Comment:
How are we checking that the invalid rects are not being rendered?

Reply:
Huh..yeah, good point; this doesn't test specifically for 0 values. Hm....thought about testing the width and height of the highlight rect but not sure if that's possible...do you have any suggestions?? :O

Reply:
`canvas` is difficult because any changes won't result in new DOM elements, so one way would be to spy on the context and see if it gets called on to draw on the canvas perhaps